packages/client/src/store/index.ts

```diff
@@ -1,8 +1,10 @@
 import { configureStore, combineReducers } from '@reduxjs/toolkit';
 import { authReducer } from './auth';
+import { registrationReducer } from './registration'
 
 const rootReducer = combineReducers({
   authReducer,
+  registrationReducer,
```

Comment:
Not a big deal but can you change these two lines to be

```
auth: authReducer,
registration: registrationReducer
```

Reply:
Fixed this 👍